verilog/tools/preprocessor/verilog_preprocessor.cc

```diff
@@ -42,25 +51,177 @@ static absl::Status StripComments(const SubcommandArgsRange& args,
     return status;
   }
 
-  // TODO(fangism): parse a subcommand-specific flag for replacement option
-  // See documentation for 'replacement' arg of the library function:
-  verilog::StripVerilogComments(source_contents, &outs, ' ');
+  char replace_char;
+  if (args.size() == 1) {
+    replace_char = ' ';
+  } else if (args.size() == 2) {
+    absl::string_view replace_str(args[1]);
+    if (replace_str.empty())
+      replace_char = '\0';
+    else if (replace_str.length() == 1)
+      replace_char = replace_str[0];
+    else
+      return absl::InvalidArgumentError(
+          "Replacement must be a single character.");
+  } else {
+    return absl::InvalidArgumentError("Too many arguments.");
+  }
+
+  verilog::StripVerilogComments(source_contents, &outs, replace_char);
 
   return absl::OkStatus();
 }
 
-static const std::pair<absl::string_view, SubcommandEntry> kCommands[] = {
-    {"strip-comments",  //
-     {&StripComments,   //
-      R"(strip-comments file
+static absl::Status PreprocessSingleFile(const char* source_file, std::istream&,
```

Comment:
Could the `source_file` be of type `std::string_view` ?

Reply:
I think it couldn't be, because `source_file` comes from `GetSubcommandEntry` which returns `std::vector<char*>`.
I mean it can be changed of course, but I will have to change `SubcommandFunction` to be a template, and maybe change something in `common/util/init_command_line.h` too. 
https://github.com/chipsalliance/verible/blob/4faba50b9fa6c30cf18b59f1254c15fc214d72c3/common/util/subcommand.h#L31-L42

Reply:
`const char*` converts naturally into a `std::string_view`, so in the loop in `MultipleCU` it will just work